test/dlc_tests/container_tests/bin/efa/efa_test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
this test gets added automatically by EFA at `/opt/amazon/efa/test/efa_test.sh`

Reply:
Summarizing offline discussion here - This script exists from EFA installation on DLAMIs, but doesn't get automatically created in DLC EFA installation. Therefore adding this script to the test scripts is necessary.